webpack.config.js

```diff
@@ -0,0 +1,30 @@
+
+const webpack = require('webpack');
+const path = require('path');
+
+const SRC_DIR = path.join(__dirname, 'client', 'src');
+const DIST_DIR = path.join(__dirname, 'client', 'public');
```

Comment:
Remember to add this path to .gitignore, otherwise, you'll commit a 20,000 line file.